client/galaxy/scripts/mvc/form/form-input.js

```diff
@@ -109,9 +109,9 @@ define([], function() {
             tmp +=              '<div class="ui-form-info">';
             if (options.help) {
                 tmp +=              options.help;
-                if (options.argument && options.help.indexOf('(' + options.argument + ')') == -1) {
-                    tmp += ' (' + options.argument + ')';
-                }
+            }
+            if (options.argument && options.help.indexOf('(' + options.argument + ')') == -1) {
+                tmp +=              ' (' + options.argument + ')';
```

Comment:
I wanted to leave it in the same style as the lines above (L111). If this is not common practice I can push the previous commit to HEAD?